media_recorder_util/video_recorder_util.cpp

```diff
@@ -0,0 +1,151 @@
+#include "video_recorder_util.h"
```

Comment:
this file with so many functions not required.
only required functionality should be there in the file.